app/assets/javascripts/3dbio_viewer/src/webapp/components/RootViewerContents.tsx

```diff
@@ -47,9 +48,21 @@ export const RootViewerContents: React.FC<RootViewerContentsProps> = React.memo(
     const toggleToolbarExpanded = React.useCallback(
         (_e: React.SyntheticEvent, data: ResizeCallbackData) => {
             setToolbarExpanded(data.size.width >= 520);
-            setViewerSelectorExpanded(window.innerWidth - data.size.width >= 725);
+            setViewerSelectorExpanded(innerWidth - data.size.width >= 725);
         },
-        []
+        [setToolbarExpanded, setViewerSelectorExpanded, innerWidth]
+    );
+
+    const resizableBoxProps = React.useMemo<
+        Pick<ResizableBoxProps, "width" | "minConstraints" | "maxConstraints" | "resizeHandles">
```

Comment:
If you apply `Required<Pick<...>>`, you'll get a more narrow type for the structure and you'll be able to directly use as props (`<ResizableBox {...resizableBoxProps}` ... />)

Reply:
Didn't know that, thanks!